File: src/Shared/Students.Models/KindOrder.cs
namespace Students.Models;

/// <summary>
/// Вид приказа
/// </summary>
public class KindOrder
{
    /// <summary>
    /// Id программы
    /// </summary>
    public Guid Id { get; set; }
    /// <summary>
    /// Имя программы
    /// </summary>
    public string Name { get; set; }
}

File: src/Client/src/storage/catalog-configs/kind-order.ts
import {
  useGetAllAsync,
  useGetAllPagedAsync,
  useGetOneByIdAsync,
  useAddOneAsync,
  useEditOneAsync,
  useRemoveOneAsync,
} from '../crud/kind-education-program-crud';
import { kindOrderModel } from '../models/index';

export default {
  detailsLink: 'kindOrder',
  hasDetailsPage: false,
  serverPaged: false,
  properties: kindOrderModel,
  crud: {
    useGetAllAsync,
    useGetAllPagedAsync,
    useGetOneByIdAsync,
    useAddOneAsync,
    useEditOneAsync,
    useRemoveOneAsync,
  },
  columns: [
    {
      title: 'Тип приказа',
      dataIndex: 'name',
      key: 'name',
    },
  ],
  dataConverter: (data) => data,
};
